config/servicemappings/cloudids.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
Please don't remove the generated yaml from autogen - it should stay there because it is automatically generated and we should avoid any manual changes like removing the file.

You should copy it here instead.

Reply:
I can call this out in the instructions doc maybe for beta ugprades

Reply:
Thank you!